plugins/woocommerce/changelog/61459-fix-61458-revert-46887

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
I think we need to bump this up to at least minor since this has been out for a couple versions now.